git_pile/cli.py

```diff
@@ -125,6 +125,17 @@ def __init__(self, config=None):
             action="version",
             version=f"git-pile {__version__}",
         )
+        self.parser.add_argument(
```

Comment:
commit message: s/respository/repository/

Reply:
Thanks! Fixed in the new version.